### PR TITLE
Wharf All-in-One chart v0.1.0

### PR DESCRIPTION
**Pull request by jilleJr**
- \[x] I've added a new note in the `charts/wharf-*/CHANGELOG.md` file, according to docs: https://iver-wharf.github.io/#/development/changelogs/writing-changelogs (but without version dates nor WIP versions)
- \[x] I've version bumped `charts/wharf-*/Chart.yml`

## Summary

- Added `wharf-aino`

## Motivation

Makes it easier to get started with Wharf.

### CODE REVIEW

File: charts/wharf-aino/values.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    config:
+      watchdog:
+        wharfApiUrl: http://wharf-aino-wharf-helm-api:80
+        provisionerUrl: http://wharf-aino-wharf-cmd-provisioner:80
+
+  aggregator:
+    config:
+      aggregator:
+        wharfApiUrl: http://wharf-aino-wharf-helm-api:80
+
```
Comment (Alexamakans) [inline charts/wharf-aino/values.yaml:28]: Doesn't this require the provisioner URL as well?

**Review by Alexamakans — APPROVED** [review]

**Comment by jilleJr** [comment]
It's failing the `install-chart` because the secret created by postgres isn't found by the wharf-api. Further investigation is needed, as it clearly does find it when testing locally.

**Comment by jilleJr** [comment]
> It's failing the `install-chart` because the secret created by postgres isn't found by the wharf-api. Further investigation is needed, as it clearly does find it when testing locally.

Idea: it's not being deployed with name "wharf-aino", so the secret has a different name

**Comment by jilleJr** [comment]
I did some digging into Helm's chart-testing (ct) util, and it does not support changing the release name (https://github.com/helm/chart-testing/blob/ab83f064a372c46938b0017a1fe4373b786e4ec0/pkg/chart/chart.go#L205-L219) There's also an existing feature request on the topic: https://github.com/helm/chart-testing/issues/310

As we cannot use `.Values...` in the `values.yaml`, we cannot set the password to be read automatically from a secret generated by the postgres chart. This is a deeper feature request on Helm, that I warn about that you do not go down that rabbit hole. There's no such feature on the horizon, so just stick with working around it.

**Review by Alexamakans — APPROVED** [review]

**Pull request merged**